core/templates/css/oppia.css

```diff
@@ -4842,3 +4844,21 @@ div.oppia-issues-learner-action {
 .cdk-global-overlay-wrapper, .cdk-overlay-container {
   z-index: 10000 !important;
 }
+
+.oppia-exploration-summary-tile {
+  margin-left: 10px;
+}
+
+.oppia-login-button {
```

Comment:
Just to confirm: are these CSS rules used in more than one component?

If they are, keeping them here is fine; just leave review comments here pointing out the components that use them (there should be at least 2).

If not then they should be renamed and scoped into their corresponding components.

(I'm assuming it's the former, given previous discussion, but just wanted to see the proof so that we do this rigorously.)

Reply:
Moved .oppia-exp-summary-tiles-container to library-page.component.ts and other styles are used in more than 1 component.